qdrant-landing/content/documentation/guides/installation.md

```diff
@@ -36,44 +104,79 @@ docker run -p 6333:6333 \
     qdrant/qdrant
 ```
 
-With this command, you will start a Qdrant instance with the default configuration.
-It will store all data in `./path/to/data` directory.
+With this command, you start a Qdrant instance with the default configuration.
+It stores all data in the `./path/to/data` directory.
 
 By default, Qdrant uses port 6333, so at [localhost:6333](http://localhost:6333) you should see the welcome message.
```

Comment:
I'm not sure whether to set this up in a separate section, such as 

`#### Docker in production

```suggestion
By default, Qdrant uses port 6333, so at [localhost:6333](http://localhost:6333) you should see the welcome message.

To set this up in production, you can then:

- Set up multiple Docker containers in a pod
- Configure the container in a Kubernetes cluster
```

Reply:
Pods is a concept introduced by Kubernetes and also used with Podman. Docker though does not use the concept of a Pod.

I'll try to come up with something.

Reply:
I tried to address this in https://github.com/qdrant/landing_page/pull/434/commits/a3a2725cccdc2072b10362dac28c0d55ba16d078